scrapeIDArtists/server.js

```diff
@@ -0,0 +1,86 @@
+const http = require('http');
+var request = require("request");
+var fs = require('fs');
+var async = require('async');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');

```

Comment:
You can run a .js file with node without having to start a http server. If you are only scraping data why do you need a webserver?